### ISSUE TITLE
Problem with loading EEG dataset and visualization in BCILAB

### ISSUE THREAD

**Issue by BCILABUser** [sccn/BCILAB#42]
Hi,

I have been trying to load my EEG data (~8 hours data from an overnight sleep study) into BCILAB for offline analyses but when I inspect it before training, the EEG plot isn't displayed correctly and chunks of my data are missing. I collected my EEG data using BrainVision Recorder and added channel locations and event labels using EEGLAB. I loaded this dataset (.set) to BCILAB. Interestingly, when I inspect the data I see 1-10 seconds data in the first window, 299.9-309.9 data in the second window, 599.8-609.8 in the third window (a consistent loss of 289.9 seconds data across windows) and so on. Moreover, the axes are wrong and it's not 'channel activities EEG plot' that I see with EEGLAB and what's shown in the BCILAB tutorial (https://www.youtube.com/watch?v=Sup2SaJyFQE&t=33). 

Does anyone have any idea why the data doesn't load properly? I tried different paradigms but I cannot train the model because of the lack of event markers in the data training because of data loss. Does anyone have any suggestions as I am very new to BCILAB? I will really appreciate any recommendations. Thanks!

**Software versions used:**
MATLAB version 2015a (Mac OS Sierra 10.12.4 but I also observed this on Windows)
BCILAB DEVEL (with EEGLAB 13_4_4b in dependencies) - With BCILAB master, I get supergui problems

**Error obtained when I try to train my data** (my event markers are present in the data when I check with EEGLAB but because of data loss, most of the events are absent in the data loaded by BCILAB)

`WARNING: the data set did not contain matching target markers.
WARNING: set_gettarget did not find any target information in this data set. See help of set_gettarget and set_targetmarkers for how data sets can be annotated with target information.

The loaded EEGLAB set is lacking an online expression; assuming it contains unfiltered data.
If it contains filtered data, however, BCI models derived from it will likely not be online-capable.
fold.
WARNING: This data set contains no trials for one of your target classes: please check whether your target marker naming is correct, and ensure that your data has enough target events in each of its required subsets (e.g., in a (nested) cross-validation).
WARNING: After set_selepos your signal is empty.
WARNING: This training set contains only one class - the subsequent learning phase will likely fail.

Computation failed; error trace:
 Your training data set has no trials for one of your classes; you need at least two classes to train a classifier.
 The most likely reasons are that one of your target markers does not occur in the data, or that all your trials of a particular class are concentrated in a single short segment of your data (10 or 20 percent). The latter would be a problem with the experiment design.
 occurred in: 
   ml_trainlda: 103
   ml_train: 170
   ParadigmDataflowSimplified.calibrate_prediction_function: 481
   ParadigmDataflowSimplified.calibrate_simple: 602
   ParadigmBaseSimplified.calibrate: 184
   instantiate_paradigm/@(varargin)instance.calibrate(varargin{:}): 0
   @(trainset,varargin)utl_complete_model(args.calibrate_func('collection',{trainset},varargin{:}),args.predict_func): 903
   evaluate_internal: 62
   cached_evaluate: 42
   utl_evaluate_fold: 34
   par_beginschedule: 166
   par_schedule: 71
   utl_crossval: 311
   utl_nested_crossval: 141
   run_computation: 938
   make_func/@(f,a,frame__f1)feval(f,a{:}): 0
   hlp_scope: 53
   bci_train: 856
   pushbutton1_Callback: 156
   gui_mainfcn: 95
   gui_calibratemodel: 30
   @(hObject,eventdata)gui_calibratemodel('pushbutton1_Callback',hObject,eventdata,guidata(hObject)): 0

`

**Comment by Wale019** [comment]
I am getting the same problem for the past 4 days..
